hack/update-codegen.sh

```diff
@@ -22,11 +22,14 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 CODEGEN_VERSION=$(cd "${KUBE_ROOT}" && grep 'k8s.io/code-generator' go.mod | awk '{print $2}')
 
 if [ -z "${GOPATH:-}" ]; then
-    export GOPATH=$(go env GOPATH)
+    GOPATH=$(go env GOPATH)
+    export GOPATH
```

Comment:
Single line with export does not work?

Reply:
Yes, It will work. It's just a best practice. For example if the go env command fails the script will fail here. But If we export in the same line the return code of the go command is masked by the export command which always succeeds. So the script won't fail. But it will eventually fail in some other line with a misleading error message.